azure/defaults.go

```diff
@@ -81,7 +81,8 @@ const (
 const (
 	// ProviderIDPrefix will be appended to the beginning of Azure resource IDs to form the Kubernetes Provider ID.
 	// NOTE: this format matches the 2 slashes format used in cloud-provider and cluster-autoscaler.
-	ProviderIDPrefix = "azure://"
+	ProviderIDPrefix          = "azure://"
+	azureBuiltInContributorID = "b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
could you please add a quick comment about what this is / where it's from? I remember you asking about it earlier and it might not be obvious to everyone